src/communication_interface.cc

```diff
@@ -409,10 +409,9 @@ void CommunicationInterface::HandlePause(
     const robot_msgs::pause_cmd* pause_cmd_msg) {
   std::lock_guard<std::mutex> lock(pause_mutex_);
   // check if paused = true or paused = false was received:
-  bool desired_pause {pause_cmd_msg->data};
   auto source {pause_cmd_msg->source};
 
-  if (desired_pause) {
+  if (pause_cmd_msg->data) {
     dexai::log()->warn(
```

Comment:
Please revert

Reply:
why? there was a narrowing conversion error and it's not used anywhere else

Reply:
We have inconsistent pause_cmd definition between `robot_msgs` and here... this needs to be fixed, it should be a bool not an int

Reply:
the definition in the `robot_msgs` has `int8_t     data;`

Reply:
🤔  I thought it was supposed to be a bool. Actually I can use this to my advantage.

Reply:
yeah it's only used as a bool here. not sure what different values represent